pipeline_tools/shared/submission/create_links.py

```diff
@@ -191,7 +191,7 @@ def main():
     parser.add_argument('--analysis_protocol_path', required=True, help='Path to the /metadata/analysis_protocol.json file.')
     parser.add_argument('--project_level', type=bool, default=False, required=False, help='Boolean representing project level vs intermediate level.')
     parser.add_argument('--workspace_version', required=True, help='A version (or timestamp) attribute shared across all workflows''within an individual workspace.')
-    parser.add_argument('--output_file_path', required=False, help='Path to the outputs.json file (This is just a json list of the /metadata/analysis_file/*.json files).')
+    parser.add_argument('--output_file_path', required=False, nargs='+', help='Path to the outputs.json file (This is just a json list of the /metadata/analysis_file/*.json files).')
```

Comment:
is there a case where there will be multiple outputs.json files? Don't think we need nargs here